src/middlewared/middlewared/plugins/nfs_/debug.py

```diff
@@ -79,6 +82,29 @@ class RPC_DBGFLAGS(enum.Enum):
     ALL = 0x7fff
 
 
+class NfsDebug(BaseModel):
+    NFS: Optional[UniqueList[NFS_DBGFLAGS.__members__]] | None = None
```

Comment:
Isn’t `Optional[…] | None` excessive?

Reply:
It is, you can remove the Optional annotation.